com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/v1/resource/impl/ModelResourceImpl.java

```diff
@@ -116,9 +116,14 @@ protected ModelResourceImpl restView() {
 		return this;
 	}
 	
-	@Override
+	@Deprecated
```

Comment:
don't have to deprecate this, please restore back the original override tag.

Reply:
Removed the deprecated annotation